LayoutTests/imported/w3c/web-platform-tests/css/css-grid/alignment/grid-align-baseline-fieldset-003-expected.txt

```diff
@@ -14,6 +14,12 @@ FAIL .target > * 2 assert_equals:
 offsetLeft expected 0 but got 5
 FAIL .target > * 3 assert_equals:
 <div data-offset-x="45"><span></span></div>
-offsetLeft expected 45 but got 25
-PASS .target > * 4
+offsetLeft expected 45 but got 0
+FAIL .target > * 4 assert_equals:
```

Comment:
Ditto: Do we expect this new failure?

Reply:
Fieldset baselines were defined in a recent conversation but I don't think they are in the spec yet: https://github.com/w3c/csswg-drafts/issues/7656

AFAIK we don't really implement this behavior so the PASS was not occurring intentionally. I created a new bug to keep track of this issue: https://bugs.webkit.org/show_bug.cgi?id=246413